### PR TITLE
Add support for hiding/showing sidebar button with custom chrome disabled

### PR DESCRIPTION
**Pull request by deathbeam**
Draw hide/show sidebar button in top-right corner using Game's draw
hook. This will not work in Vanilla and so it is disabled in it.

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

screenshot 2018-05-25 18 30 38

peek 2018-06-06 10-34

This is extracted from #2872

### CODE REVIEW

**Comment by Nightfirecat** [comment]
Why are the SwingUtil changes included? It doesn't even seem to be used in the sidebar arrow code.

**Comment by Nightfirecat** [comment]
Also, how does this look in fixed mode with the FPS plugin enabled?

**Comment by deathbeam** [comment]
I was using that before, I saw unused method, I removed it. And, in fixed mode it looks exactly like in resizable mode with FPS plugin enabled

File: runelite-client/src/main/java/net/runelite/client/util/SwingUtil.java [hunk 1]
```diff
@@ -186,20 +183,6 @@ public void mouseClicked(MouseEvent e)
...
 	}
 
-	/**
-	 * Check if point is in screen bounds.
-	 *
-	 * @param x the x
-	 * @param y the y
-	 * @return the boolean
-	 */
-	public static boolean isInScreenBounds(final int x, final int y)
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/util/SwingUtil.java]: As I understand it, the changes in this file were from a cherry-picked commit from the "Remove custom UI" PR, but I'd like to see them be removed or in a separate commit since they aren't used in the substantive part of this change.
